alg/gdaltransformer.cpp

```diff
@@ -601,13 +588,15 @@ CPLErr CPL_STDCALL GDALSuggestedWarpOutput2(GDALDatasetH hSrcDS,
                     double ayTemp[2] = {padfY[i], padfY[i]};
                     double azTemp[2] = {padfZ[i], padfZ[i]};
                     int abSuccess[2] = {FALSE, FALSE};
+                    CPLTurnFailureIntoWarning(true);
```

Comment:
side note: should we have a context manager for this settings temporary changes?

Reply:
good idea. I had the same thought too when writing that code. ticketed as https://github.com/OSGeo/gdal/issues/11831